operations/Makefile

```diff
@@ -1,16 +1,24 @@
 SHELL := bash
 .SHELLFLAGS := -eu -o pipefail -c
 
-TF_ENV = "dev"
-TF_ENV_MODULE = ""
+TF_ENV = "dev" #VAR Your target environment (e.g. dev, test, staging, prod)
+TF_STAGE = ""  # not surfaced
```

Comment:
Thank you for documenting! `TF_STAGE` is logical. I always struggle with variable naming, haha. Seeing `TF_STAGE` now, I'm like: "Doh! That makes significantly more sense!"